lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala

```diff
@@ -493,6 +493,11 @@ private[scalacoptions] trait ScalacOptions {
   val privateRetainTrees =
     privateOption("retain-trees", version => version >= V3_0_0)
 
+  /** Enables retain trees for top-level classes, accessible from ClassSymbol#tree
```

Comment:
This needs correcting. Copied from above entry.

Reply:
Thanks. Pushed a good description